dev/bots/test.dart

```diff
@@ -519,7 +519,7 @@ Future<void> _flutterBuild(
     final File file = File(path.join(flutterRoot, relativePathToApplication, 'perf.json'));
     if (!_allTargetsCached(file)) {
       print('${red}Not all build targets cached after second run.$reset');
-      print('The target performance data was: ${file.readAsStringSync()}');
+      print('The target performance data was: ${file.readAsStringSync().replaceAll('},', '},\n')}');
```

Comment:
Makes the log much more readable. Quick fix for me totally missing the "skipped: false" entries in this line because it was 90% cut off onscreen.

Reply:
awesome!